jaxrs-spec/src/main/asciidoc/chapters/client_api/_client_exceptions.adoc

```diff
@@ -0,0 +1,32 @@
+////
+*******************************************************************
+* Copyright (c) 2022 Eclipse Foundation
+*
+* This specification document is made available under the terms
+* of the Eclipse Foundation Specification License v1.0, which is
+* available at https://www.eclipse.org/legal/efsl.php.
+*******************************************************************
+////
+
+[[client_exceptions]]
+=== Client Exceptions
+
+A method of the JAX-RS Client API that _nominally_ throws `WebApplicationException`
+MUST _actually_ throw that specific subclass of `WebApplicationException` defined by
+the JAX-RS API, that is _at most specific_ the current response's status code,
+unless no such subclass is defined by the JAX-RS API.
+
```

Comment:
I would just simplify the paragraph structure a bit. "A method of the JAX-RS Client API that _nominally_ throws `WebApplicationException` MUST _actually_ throw the most specific exception subclass for the current response status code, unless no such subclass is defined by the JAX-RS API.